db/re/item_db.conf

```diff
@@ -42242,8 +42307,9 @@ item_db: (
 	Weight: 10
 	Loc: 2
 	Script: <"
-		bonus2 bSkillAtk,GC_CROSSIMPACT,100;
-		if (getrefine() >= 10) bonus2 bSkillAtk,GC_CROSSIMPACT,200;
+		bonus2 bSkillAtk,GC_CROSSIMPACT,50;
+		if (getequipid(EQI_HAND_R) == 4) bonus2 bSkillAtk,GC_CROSSIMPACT,50;
```

Comment:
`if (getequipid(EQI_HAND_R) == 4)`
means, if the right hand equipment item ID is 4
is there any Item ID 4 ??
I thought it start from 501, and its Red_Potion !

something wrong on this line